### PR TITLE
#2911 Add all-scroll for selected structure

### PR DESCRIPTION
**Pull request by aproskurnov**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [x] unit-tests written
- [x] e2e-tests written
- [x] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"

### CODE REVIEW

File: ketcher-autotests/tests/utils/canvas/helpers.ts [hunk 1]
```diff
@@ -29,8 +29,9 @@ export async function drawElementByTitle(
 }
 
 export async function getLeftToolBarWidth(page: Page): Promise<number> {
-  const leftBar = await page.locator('[class^="LeftToolbar-module_root"]');
-  const leftBarSize = await leftBar.boundingBox();
+  const leftBarSize = await page
+    .getByTestId('LeftToolbar-module_root')
```
Comment (Nitvex) [inline ketcher-autotests/tests/utils/canvas/helpers.ts:33]: I think this kind of classes was generated by some library and my proposal is to have more meaningful names. May be something like `.getByTestId('left-toolbar')`

File: packages/ketcher-react/src/script/editor/tool/select.ts [hunk 2]
```diff
@@ -443,6 +452,7 @@ class SelectTool implements Tool {
   }
 
   mouseleave() {
+    console.log('leave');
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/editor/tool/select.ts:455]: Guess, you need to remove this :)

**Review by SashaGraves — APPROVED** [review]

**Comment by Zhirnoff** [comment]
Tested
2023-07-31_18h12_20

File: ketcher-autotests/tests/utils/canvas/helpers.ts (hunk 1 above)
Comment (Zhirnoff) [reply ketcher-autotests/tests/utils/canvas/helpers.ts:33]: I agree with this comment

**Review by Nitvex — APPROVED** [review]

File: packages/ketcher-react/src/script/editor/tool/select.ts (hunk 2 above)
Comment (aproskurnov) [reply packages/ketcher-react/src/script/editor/tool/select.ts]: fixed

**Pull request merged**